### PR TITLE
Revert using one line macro

### PR DESCRIPTION
**Pull request by stevenjoezhang**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes was maked (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [ ] [Docs](https://github.com/theme-next/theme-next.org/tree/source/source/docs) in [NexT website](https://theme-next.org/docs/) have been added / updated (for features).

## PR Type
**What kind of change does this PR introduce?**

- [ ] Bugfix.
- [ ] Feature.
- [x] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue resolved: N/A

## What is the new behavior?

- Screenshots with this changes: N/A
- Link to demo site with this changes: N/A

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by 1v9** [comment]
Something wrong?

**Comment by stevenjoezhang** [comment]
Just for code-style unification...

File: source/css/_mixins/base.styl [hunk 1]
```diff
@@ -64,15 +64,15 @@ desktop-largest() {
...
 
-hide() {
+display: none {
   display: none;
 }
 
-show() {
+display: block {
   display: block;
 }
```
Comment (ivan-nginx) [inline source/css/_mixins/base.styl]: So, why need that repeated variables?

Comment (1v9) [reply source/css/_mixins/base.styl]: It's funny 😂

Comment (stevenjoezhang) [reply source/css/_mixins/base.styl]: It looks like I made a mistake with the substitution, but fortunately it didn't cause a syntax error... Stylint doesn't report this either: https://github.com/theme-next/hexo-theme-next/pull/1037

Comment (stevenjoezhang) [reply source/css/_mixins/base.styl]: Fixed in https://github.com/theme-next/hexo-theme-next/commit/07b05049193c34e58361fb515ed8d8fafa9ddb75

**Pull request merged**